src/test/java/org/fundacionjala/pivotal/steps/api/APIProjectSteps.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.pivotal.steps.api;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import org.fundacionjala.pivotal.api.APIProjectsRequests;
+import org.testng.Assert;
+import java.util.Map;
+
+/**
+ * APIProjectSteps.java
+ * Class with steps for test feature.
+ */
+public class APIProjectSteps {
+
+    public static final String PROJECT_ENDPOINT = "/projects/";
+    private APIProjectsRequests requests;
+
+    /**
+     * Given.
+     * @param values received as maps.
+     */
+    @Given("^I post a new project$")
+    public void iPostANewProject(final Map<String, String> values) {
+        requests = new APIProjectsRequests();
+        requests.setBaseUri("https://www.pivotaltracker.com/services/v5");
+        requests.postNewProject(values, PROJECT_ENDPOINT);
+    }
+
+    /**
+     * Then, when a post is made.
+     */
+    @Then("^I get body of the answer$")
+    public void iGetBodyOfTheAnswer() {
+        final int expectedStatus = 200;
+        Assert.assertEquals(requests.getStatusCode(), expectedStatus);
+        requests.getResponseBody();
+    }
+
+    /**
+     * Given.
+     * @param values map.
+     */
+    @Given("^I delete a project$")
+    public void iDeleteAProject(final Map<String, String> values) {
+        requests = new APIProjectsRequests();
+        requests.setBaseUri("https://www.pivotaltracker.com/services/v5");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)